lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -656,7 +656,7 @@ public function get_item_schema() {
 				'slug'        => array(
 					'description' => 'An alphanumeric identifier for the object unique to its type.',
 					'type'        => 'string',
-					'context'     => array( 'embed', 'view', 'edit' ),
+					'context'     => array( 'view', 'edit' ),
```

Comment:
@danielbachhuber Is there a use case for having the Users slug without authentication?  Keeping in mind that `get_author_posts_url()` is filterable to not use `user_nicename` (or `slug`)